lib/Extractor/Solver.cpp

```diff
@@ -111,6 +115,13 @@ class BaseSolver : public Solver {
       }
     }
 
+    if (InferInsts && SMTSolver->supportsModels()) {
+      InstSynthesis IS;
+      EC = IS.synthesize(SMTSolver.get(), BPCs, PCs, LHS, RHS, IC, Timeout);
+      if (EC || RHS)
```

Comment:
Thanks, I ran into a bug recently too and now I'm setting RHS to 0 explicitly since then.